implementation/choreography-to-orchestration/lambda/orchestration/ShipOrder.py

```diff
@@ -0,0 +1,9 @@
+import json
+
+def lambda_handler(event, context):
+    # TODO implement
+    print ('Shipped order for the product id:', event['product_id'])
```

Comment:
Same here. Per your diagram, patterns/images/choreography-to-orchestration/orchestration.png, the Lambda stores data. Looks like it is missing here

Reply:
Added Dynamodb table